apps/conversations/models.py

```diff
@@ -166,3 +196,21 @@ def save(self, **kwargs):
 
         # force update on `Conversation.updated_at`
         self.conversation.save()
+
+
+class BaseConversationMessageReaction(models.Model):
+    """An emoji reaction to a message in
+    a conversation.
+    """
+
+    message = fk_conversation_message(related_name='reactions', required=True)
+    user = fk_user(related_name='reacted_by', required=False)
```

Comment:
```suggestion
    user = fk_user(related_name='reactions', required=False)
```

This should be from the perspective of the object. `user.reactions` or `user.message_reactions` would be preferred, because `user.reacted_by` does not make sense.

Reply:
Oh right, I should have been more careful of the foreign key names. I understand the `related_name` field and won't make this mistake again.